code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+	affecting = null
+	REMOVE_TRAIT(src, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_helmet)//Should be attached, might not be attached.
+		REMOVE_TRAIT(c_helmet, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_shoes)
+		REMOVE_TRAIT(c_shoes, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+	if(c_gauntlets)
+		REMOVE_TRAIT(c_gauntlets, TRAIT_NODROP, CLOCK_SUIT_TRAIT)
+
+// Initialize messages
+
+#define CLOCK_LOCK_PHASE 				1
+#define CLOCK_COMPLETE_PHASE	 		6
+#define CLOCK_DEINIT_LOGOFF_PHASE		1
+
+GLOBAL_LIST_INIT(clock_initialize_messages, list(
+	"Redirecting the remains of the Justiciar...",
+	"Securing hardsuit...\nConnection established.",
+	"Finding Reebee...",
+	"Connection with Reebee failed.",
+	"Siphoning remains of power into suit...",
+	"Ratvar's last efforts located.",
+	"All systems operational, "
+))
+
+GLOBAL_LIST_INIT(clock_deinitialize_messages, list(
+	"Now de-initializing...",
+	"Shutting down connection with the <B>Justiciar</B>.",
+	"Using the Justiciar's spare power...",
+	"Repelling Repulsive Nar'Sian influences...",
+	"Purifying suit from unholy influences.",
+	"Purification complete.",
+	"Unsecuring hardsuit mechanism."
+))
+
+
+/obj/item/clothing/suit/space/clockwork/proc/toggle_on_off()
+	. = TRUE
+	if(s_busy)
+		to_chat(loc, "<span class='warning'/span> The Justiciar's power must wait.")
+		return FALSE
+	s_busy = TRUE
+	if(s_initialized)
+		deinitialize()
+	else
+		cinitialize()
+
+/obj/item/clothing/suit/space/clockwork/ui_action_click(mob/user, action)
+	if(istype(action, /datum/action/item_action/initialize_clock_suit))
+		toggle_on_off()
+		return TRUE
+
+/obj/item/clothing/suit/space/clockwork/proc/cinitialize(delay = s_delay, mob/living/carbon/human/U = loc, phase = 0)
+	if(!U || !U.mind)
+		s_busy = FALSE
+		return
+	if (phase > CLOCK_LOCK_PHASE && (U.stat == DEAD || U.health <= 0))
+		to_chat(U, "<span class='danger'><B>ơ͙͈̻n̵̘̦l̫̬̀ͅyͅ ̲ò̮̤n̼̻̼̖̭̪͠ͅe̬͚͕̲̻̰ ҉͚̞͙̞̭̼s̡͖h̜̻̞̣̩̟ͅa̴͉͍̯̟͎̠l̳̝̖̫͞ḷ͔̘̞͞ ̯̞̥̱̲͙̕ͅe̛̙m̦̻̗̹̝̹͘e͈r̫g͕͉̠e͢ ̮͎̗̮̭̭̝a̱͍͍ͅl͙̗͔̹̻̗͈i͕͕̣̫̳̞v̗͈͍͝e</B>F̶o̷r̸ ̶t̸h̷e̸ ̴J̵u̴s̷t̴i̶c̸i̴a̷r̶-̶!̸!̶̨͖̭̜̱̼̺̘̙͇̦̲̬̱͊̏͒͛̍</span>")
+		unlock_suit()
+		s_busy = FALSE
+		return
+
+	var/message = GLOB.clock_initialize_messages[phase + 1]
+	switch(phase)
+		if (CLOCK_LOCK_PHASE)
+			if(!lock_suit(U))//To lock the suit onto wearer.
+				s_busy = FALSE
+				return
+		if (CLOCK_COMPLETE_PHASE)
+			message += "Make the Justiciar proud, [U.real_name]."
+			s_initialized = TRUE
+			s_busy = FALSE
+
+	to_chat(U, "<span class='notice'>[message]</span>")
+	playsound(U, 'sound/effects/sparks1.ogg', 10, TRUE)
+
+	if (phase < CLOCK_COMPLETE_PHASE)
+		addtimer(CALLBACK(src, .proc/cinitialize, delay, U, phase + 1), delay)
+
+/obj/item/clothing/suit/space/clockwork/proc/deinitialize(delay = s_delay, mob/living/carbon/human/U = affecting == loc ? affecting : null, phase = 0)
+	if (!U || !U.mind)
+		s_busy = FALSE
+		return
+	if (phase == 0 && alert("Do you really wish to drive away the Justiciar's remaining power?",,"Yes","No") == "No")
```

Comment:
is the ",," here intentional?

Reply:
also, you can stall this alert to make this suit not count as being "busy" at a later point of your choosing